pandas/core/generic.py

```diff
@@ -5201,60 +5201,217 @@ def abs(self):
         """
         return np.abs(self)
 
-    _shared_docs['describe'] = """
-        Generate various summary statistics, excluding NaN values.
+    def describe(self, percentiles=None, include=None, exclude=None):
+        """
+        Generates descriptive statistics that summarize the central tendency,
+        dispersion and shape of a dataset's distribution, excluding
+        ``NaN`` values.
 
```

Comment:
I would add here the sentence from the notes with something like "Analyzes both numeric and object series, as well
as DataFrame column sets of mixed data types." + that output depends on data type + refer to notes for more details on this